documentation/introduction/gettingStarted/installation/development.md

```diff
@@ -26,17 +26,18 @@ npm run serve
 
 ### Backend
 
-And finally, in a new terminal, run the backend server.
+And finally, in a new terminal, run the debiai server.
 
 ```bash
-cd debiai/backend
+cd debiai/debiaiServer
 pip install -r requirements.txt
-python3 websrv.py
+cd debiai
+python3 run_debiai_server_dev.py
 ```
```

Comment:
cd debiai/debiaiServer
pip install -r requirements.txt
cd debiai
python3 run_debiai_server_dev.py

`debiai not found`